sigar_windows.go

```diff
@@ -306,6 +312,45 @@ func GetProcName(pid int) (string, error) {
 
 }
 
+func GetProcCredName(pid int) (string, error) {
+	handle, err := syscall.OpenProcess(syscall.PROCESS_QUERY_INFORMATION, false, uint32(pid))
+
+	defer syscall.CloseHandle(handle)
+	
+	if err != nil {
+		return "", fmt.Errorf("OpenProcess fails with %v", err)
+	}
+	
+	var token syscall.Token
+	
+	// Find process token via win32
+	retOpenProcessToken := syscall.OpenProcessToken(handle, syscall.TOKEN_QUERY, &token)
```

Comment:
Close the access token handle returned through the TokenHandle parameter of OpenProcessToken to prevent a handle leak.